app/src/main/java/org/fossasia/openevent/app/data/contract/ILoginModel.java

```diff
@@ -1,10 +1,10 @@
-package org.fossasia.openevent.app.data;
+package org.fossasia.openevent.app.data.contract;
 
 import org.fossasia.openevent.app.data.models.LoginResponse;
 
 import io.reactivex.Observable;
 
-public interface ILoginDataRepository {
+public interface ILoginModel {
```

Comment:
MVP pattern means Model, View, Presenter 

Model means container of any kind of interface with which presenter interacts, so by that means every model can be confused with data model

It's just a model for performing login, definitely not a repository